applications/luci-app-keepalived/htdocs/luci-static/resources/view/keepalived/vrrp_instance.js

```diff
@@ -0,0 +1,286 @@
+'use strict';
+'require view';
+'require form';
+'require uci';
+'require network';
+'require tools.widgets as widgets';
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			network.getDevices(),
+			uci.load('keepalived'),
+		]);
+	},
+
+	renderGeneralTab: function(s) {
+		var o, ipaddress;
+
+		o = s.taboption('general',form.Value, 'name', _('Name'));
+		o.rmempty = false;
+		o.optional = false;
+		o.placeholder = 'name';
+
+		o = s.taboption('general', form.Value, 'state', _('State'),
+			_('Initial State. As soon as the other machine(s) come up,') +
+			_('an election will be held and the machine with the highest "priority" will become MASTER.'));
+		o.value('MASTER', _('Master'));
+		o.value('BACKUP', _('Backup'));
+		o.optional = false;
+		o.rmempty = false;
+
+		o = s.taboption('general', widgets.DeviceSelect, 'interface', _('Interface'),
+			_('Interface for inside_network, bound by VRRP'));
+		o.noaliases = true;
+		o.noinactive = true;
+		o.optional = false;
+		o.rmempty = false;
+
+		o = s.taboption('general', form.Value, 'virtual_router_id', _('Virtual Router Id'),
+			_('Differentiate multiple instances of vrrpd, running on the same NIC'));
+		o.datatype = 'range(1-255)';
+		o.optional = false;
+		o.rmempty = false;
+
+		o = s.taboption('general', form.Value, 'priority', _('Priority'),
+			_('A server with a higher priority becomes a MASTER'));
+		o.datatype = 'uinteger';
+		o.optional = false;
+		o.rmempty = false;
+
+		o = s.taboption('general', form.Value, 'advert_int', _('Interval'),
+			_('VRRP Advert interval in seconds'));
+		o.datatype = 'float';
+		o.default = '1';
+		o.rmempty = false;
+		o.optional = false;
+		o.placeholder = '1';
```

Comment:
Remove this line and add some common seconds as values, so we have a dropdown

Reply:
Done